### ISSUE TITLE
About Pre-train stage

### ISSUE THREAD

**Issue by qfchou** [ls1rius/WSI_FiVE#4]
Hi,

Thank you for sharing your excellent work.

I wanted to ask if you could provide instructions on how to pre-train this method for zero-shot settings?  I am able to see the provided trained weights and log files but I could not find any instructions/config-files to re-train from scratch for zero-shot experiments.

Thank you in advance and looking forward your answers!

**Comment by ls1rius** [comment]
The train method written in the readme is zero-shot setting. The model is trained using the wsi-report data, and the TCGA label is utilized for zero-shot prediction.

**Comment by qfchou** [comment]
Thank you!

Now I want to reproduce the pre-train stage with wsi-report data. Can you provide any instructions/config-files to pre-train the model with wsi-report data from scratch for zero-shot and supervised experiments? I can see your pth file but now I want to train it from scratch.

**Comment by ls1rius** [comment]
Just follow the 'train' method in the readme.

**Comment by qfchou** [comment]
So the 'train' method in the readme means training the model with the wsi-report data and do zero-shot experiment with this model right? Then how can I do supervised experiment with the pre-trained model?

**Comment by ls1rius** [comment]
Yes.
using fix_pth.yaml to pretrain
using fix_pth_ft.yaml and load the pretrained pth to finetune

**Comment by qfchou** [comment]
Thank you for your patience!
